msi/rust.wxs

```diff
@@ -192,9 +201,9 @@
                         <Shortcut Id="RustShell"
                                   Name="$(var.ProductName) Shell"
                                   Description="Opens Command Prompt with Rust tools directory added to the PATH"
-                                  Target="[SystemFolder]CMD.exe"
+                                  Target="[SHORTCUTTARGET]"
```

Comment:
`SystemFolder` resolves to `C:\Windows\SysWow64` on 64-bit Windows which contains the 32-bit `cmd.exe`. I picked `%windir%\System32` because that's what the shortcut to Command Prompt in the start menu uses.